.storybook/preview.js

```diff
@@ -216,12 +229,17 @@ Vue.use(VueI18Next)
 const { $themePath } = require('app/core/initialize-themes')
 Vue.prototype.$themePath = $themePath
 
+import api from 'core/api'
+
+
 export const decorators = [
   (() => {
     let oldContext = null;
 
     return (storyFn, context) => {
 
+      api.trialRequests = {getOwn: () => Promise.resolve({ data: 'mockedData' })};
```

Comment:
It's been added to prevent errors in storybook, because `ModalGetLicenses` calls for trialReuests, but we don't have api in storybook, so it failed. 
We don't actually need the `data: 'mockedData'`, a `getOwn` function that doesn't go on error.